main/src/library/Applicative.flix

```diff
@@ -50,28 +50,28 @@ pub lawful class Applicative[m : Type -> Type] with Functor[m] {
     /// Instances can define more efficient implementations than the default implementation
     /// (which is `Applicative.ap(Functor.map(f, x1), x2)`).
     ///
-    pub def liftA2(f: t1 -> t2 -> r \ ef, x1: m[t1], x2: m[t2]): m[r] \ ef = Applicative.ap(Functor.map(f, x1), x2)
+    pub def mapA2(f: t1 -> t2 -> r \ ef, x1: m[t1], x2: m[t2]): m[r] \ ef = Applicative.ap(Functor.map(f, x1), x2)
```

Comment:
I think these should be `map2` without the `A` (at least to follow the Scala cats naming we are using).

Reply:
Right - will fix.

Reply:
There's also a `lift` function in `Functor` that is just `Functor.map`.

Do we want to remove it - or look at it another time?

Finally `Functor.as` shares its name with a keyword - would it be better called `Functor.replace`?

Reply:
Yes to both i think

Reply:
Both are done in the latest commit.